tests/setup-polly.ts

```diff
@@ -1,18 +1,79 @@
 import * as path from 'path';
-
-import replaceAll from 'string.prototype.replaceall';
-
 // @ts-ignore
 import { setupPolly } from 'setup-polly-jest';
-
 // @ts-ignore
 import FSPersister from '@pollyjs/persister-fs';
+import { Har, Header } from 'har-format';
+
+type HeaderDefaults = {
+  [key: string]: string;
+};
```

Comment:
Can we use [`Record`](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeystype) here?

```suggestion
type HeaderDefaults = Record<string, string>;
```

Reply:
👍 done!